.pre-commit-config.yaml

```
@@ -0,0 +1,81 @@
+      - id: check-added-large-files
+        args: ['--maxkb=500']
+      - id: check-case-conflict
+      - id: check-merge-conflict
+      - id: check-json
+      - id: check-toml
+      - id: check-xml
+      - id: mixed-line-ending
+        args: ['--fix=lf']
+      - id: no-commit-to-branch
+        args: ['--branch', 'main', '--branch', 'master']
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)